protocol/src/test/java/org/fido/iot/protocol/To2AsymkexTest.java

```diff
@@ -0,0 +1,374 @@
+    fromOwnerInfo = null;
+
+    super.setup();
+
+    final To2ClientStorage clientStorage = new To2ClientStorage() {
+
+      @Override
+      public PrivateKey getSigningKey() {
+        return PemLoader.loadPrivateKey(devKeyPem);
+      }
+
+      @Override
+      public Composite getSigInfoA() {
+        return cryptoService
+            .getSignInfo(
+                PemLoader.loadCerts(devKeyPem)
+                    .get(0)
+                    .getPublicKey());
+      }
+
+      @Override
+      public byte[] getMaroePrefix() {
+        return null;
+      }
+
+      @Override
+      public String getKexSuiteName() {
+        return Const.ASYMKEX2048_ALG_NAME;
+      }
+
+      @Override
+      public String getCipherSuiteName() {
+        return Const.AES128_CTR_HMAC256_ALG_NAME;
+      }
+
+      @Override
+      public byte[] getReplacementHmacSecret(Composite newCredentials, boolean isReuse) {
+        return null;
+      }
+
+      @Override
+      public void prepareServiceInfo() {
+        Composite value = ServiceInfoEncoder.encodeValue("devmod:active", "true");
+        List<Composite> list = new ArrayList<>();
+        list.add(value);
+        toOwnerInfo = ServiceInfoEncoder.encodeDeviceServiceInfo(list, false);
+      }
+
+      @Override
+      public Composite getNextServiceInfo() {
+        Composite result = toOwnerInfo;
+        toOwnerInfo = ServiceInfoEncoder.encodeDeviceServiceInfo(
+            Collections.EMPTY_LIST, false);
+        return result;
+      }
+
+      @Override
+      public void setServiceInfo(Composite info, boolean isMore, boolean isDone) {
+        fromOwnerInfo = info;
+      }
+
+      @Override
+      public Composite getDeviceCredentials() {
+        return deviceCreds;
+      }
+
+      @Override
+      public void starting(Composite request, Composite reply) {
+
+      }
+
+      @Override
+      public void started(Composite request, Composite reply) {
+
+      }
+
+      @Override
+      public void continuing(Composite request, Composite reply) {
+        Composite info = request.getAsComposite(Const.SM_PROTOCOL_INFO);
+        if (info.containsKey(Const.PI_TOKEN)) {
+          clientToken = info.getAsString(Const.PI_TOKEN);
+        }
+        reply.set(Const.SM_PROTOCOL_INFO,
+            Composite.newMap().set(Const.PI_TOKEN, clientToken));
+      }
+
+      @Override
+      public void continued(Composite request, Composite reply) {
+
+      }
+
+      @Override
+      public void completed(Composite request, Composite reply) {
+
+      }
+
+      @Override
+      public void failed(Composite request, Composite reply) {
+
+      }
+    };
+
+    clientService = new To2ClientService() {
+      @Override
+      protected To2ClientStorage getStorage() {
+        return clientStorage;
+      }
+
+      @Override
+      public CryptoService getCryptoService() {
+        return cryptoService;
+      }
+    };
+
+    To2ServerStorage serverStorage = new To2ServerStorage() {
```

Comment:
To2ServerStorage interface has been updated in the merger [PR#21](https://github.com/secure-device-onboard/pri-fidoiot/pull/21), which is why the build is failing. Please update this as per the updated interface.